src/main/docker/vpn/client/Dockerfile

```diff
@@ -1,16 +1,15 @@
-FROM ubuntu:14.04
+FROM alpine
```

Comment:
For a static image, we should probably test with `alpine:3.4`
See the latest stable version here: https://hub.docker.com/_/alpine/